container_files/public_html/doc/builtin/sql/ValueExpression.md

```diff
@@ -684,3 +684,30 @@ log to the console to aid debugging. Documentation for this object can be found
 
 You can also take a look at the ![](%%nblink _tutorials/Executing JavaScript Code Directly in SQL Queries Using the jseval Function Tutorial) for examples of how to use the `jseval` function.
 
+## <a name="try"></a>Handling errors line by line
+
+When a function is given invalid input, the whole query fails and no result is
+returned. The `try` function is meant to handle those cases. The first argument
+is the expression you want to "try" to apply. The optional second argument is
+what will be done if an error is encountered. It can be any value expression
+including other functions and other `try`. If no second argument is given, the
+error is returned.
+
+Example usage
+
+```sql
+SELECT try(parse_json('foo'), 'err')
+```
+
+Here, `parse_json('foo')` will fail. Since the second argument is provided, the
+value "err" will be returned..
+
+```sql
+SELECT try(parse_json('foo'))
+```
+
+Again, `parse_json('foo')` will fail. Since the second argument was left blank,
+the error message generated by MLDB will be returned.
```

Comment:
nouveua paragraphe.

If the first argument is expected to return a row and bla bla bla, then the second argument should also be a row expression (eg.: an empty row `{}`).